src/components/message-input/styles.scss

```diff
@@ -27,242 +27,182 @@
+    justify-content: center;
+    height: 32px;
+    width: 32px;
+    color: theme.$color-greyscale-12;
 
-.message {
-  &__block-body {
-    .message-input__input-wrapper {
-      background: none;
-      &::before {
-        background: none;
-      }
+    &--send {
+      color: theme.$color-secondary-11;
     }
-  }
-}
-
-.mentions-text-area__wrap {
-  position: relative;
-  overflow-y: visible;
-  &__highlighter {
-    position: relative;
-    box-sizing: border-box;
-    width: 100%;
-    color: transparent;
-    overflow: hidden;
-    white-space: pre-wrap;
-    background: transparent;
-    overflow-wrap: break-word;
-    border: 1px solid transparent;
-    text-align: start;
 
-    &__substring {
-      visibility: hidden;
+    &--emoji {
+      color: theme.$color-greyscale-11;
     }
-
-    @include tag-highlight;
   }
 
-  &__input {
-    display: block;
-    width: 100%;
-    position: absolute;
-    margin: 0px;
-    top: 0px;
-    left: 0px;
-    box-sizing: border-box;
-    background-color: transparent;
-    font-family: inherit;
-    font-size: inherit;
-    letter-spacing: inherit;
-    height: 100%;
-    bottom: 0px;
-    overflow: hidden;
-    resize: none;
-  }
-
-  &__suggestions {
-    background: none !important;
-    z-index: 1000 !important;
-    animation: fadein animation.$animation-duration-double ease-in-out;
-    &__list {
-      border: 2px solid #464646;
-      background-color: themeDeprecated.$background-color-primary-selected;
-    }
-    &__item {
-      padding: 5px 10px;
-      &--focused {
-        background-color: #141414;
-      }
-      &__display {
-        color: #cccccc;
-      }
-      &__highlight {
-        font-weight: bold;
-        color: white;
-      }
-    }
+  &__emoji-icon-outer {
+    display: flex;
+    align-self: flex-end;
   }
-}
 
-.message-input {
-  &__icons {
+  &__chat-container {
     display: flex;
-    align-items: center;
+    padding: 8px 8px 8px 16px;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-end;
     gap: 8px;
+    flex: 1 0 0;
 
-    &-action {
-      position: absolute;
-      right: 10px;
-      bottom: 8px;
-      z-index: 1;
-      color: theme.$color-greyscale-11;
+    background-color: theme.$color-primary-1;
+    border: 1px solid theme.$color-greyscale-6;
+    border-radius: 8px;
+
+    &:focus-within {
+      border-color: theme.$color-primary-7;
     }
   }
 
-  &__giphy-icon {
+  &__text-and-emoji-wrapper {
     display: flex;
+    justify-content: flex-end;
     align-items: center;
-    justify-content: center;
-    height: 32px;
-    width: 32px;
+    gap: 8px;
+    flex-shrink: 0;
+    align-self: stretch;
   }
 
-  &__record-voice-icon {
+  &__mentions-text-area {
     display: flex;
-    align-items: center;
-    justify-content: center;
-    height: 32px;
-    width: 32px;
+    flex-direction: column;
+    flex: 1 0 0;
+    font-size: 14px;
+    line-height: 17px;
+
+    &__wrap {
+      &__suggestions {
```

Comment:
why not use `&__wrap-suggestions` here, if it's directly a child of the same class with no additional parent css?

Reply:
Ha good question. I couldn't get this working that way round, very odd! For the time it was worth, I just left it as is for now.